src/http/one/ResponseParser.cc

```diff
@@ -18,11 +18,11 @@
 const SBuf Http::One::ResponseParser::IcyMagic("ICY ");
 
 Http1::Parser::size_type
-Http::One::ResponseParser::firstLineSize() const
+Http::One::ResponseParser::StatusLineSize(const AnyP::ProtocolVersion &msgProtocol, const size_t reasonLength)
```

Comment:
Regarding the name change: To avoid duplication of this non-trivial code, we needed to convert this method into a static function member. Long-term, this non-parsing code should not be inside a (parser) class at all -- it should be a free function -- but we could not move it outside in this PR because the method implementation uses some (needlessly) protected constants from the parent class. When the parent class code is adjusted to avoid hiding those constants, this should become a free function. And that free function will use this response-specific name to avoid clashing with a similar request-specific function.